habitat-hitl/README.md

```diff
@@ -43,6 +43,14 @@ Example HITL apps are configured to run at 30 steps per second (SPS). If your sy
     --data-path data/
     ```
 
+## Run Requirements
```

Comment:
I took at stab at rewriting this:

## Data directory and running location

HITL apps (and Habitat libraries in general) expect a `data/` directory in the running location (aka current working directory). Notice the `--data-path` argument in our installation steps above. Here are two options to consider:

1. Download data to `habitat-lab/data` as shown in our installation steps above. This is the default location for many Habitat tutorials and utilities. Run your HITL app from this location, e.g. `habitat-lab/$ python /path/to/my_hitl_app/my_hitl_app.py`.
2. Download (or symlink) data to your HITL app's root directory, e.g. `/path/to/my_hitl_app/data`. Run your HITL app from this location, e.g. `/path/to/my_hitl_app/$ python my_hitl_app.py`."

Reply:
In addition, we should update step 5 in our installation steps to clarify that this command (the dataset downloader) should be run from `habitat-lab/`.